faucet-server/Sources/App/Services/Authentication.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  AuthorizationService.swift
+//  App
+//
+//  Created by 翟泉 on 2019/3/12.
+//
+
+import Foundation
+import Vapor
+
+class Authentication {
+    enum Status: Int {
+        case tokenIsVailable = 0
+        case unauthenticated = -1
+        case received = -2
+    }
+
+    func verify(email: String, on connection: Request) -> EventLoopFuture<Status> {
+        return User
+            .query(on: connection)
+            .filter(\.email, .equal, email)
+            .first()
+            .map { user -> Status in
+            if let user = user {
+                if user.recentlyReceivedDate?.timeIntervalSince1970 ?? 0 < Date().timeIntervalSince1970 - 24 * 60 * 60 {
+                    return .tokenIsVailable
```

Comment:
`tokenIsVailable` seems to be a name having typo.

Reply:
https://github.com/nervosnetwork/ckb-testnet-faucet/pull/85/commits/28e042d709ada331068461571cea010bb113037e